lib/msal-angular/src/msal.broadcast.service.spec.ts

```diff
@@ -291,9 +290,9 @@ describe('MsalBroadcastService', () => {
     });
 
     // @ts-ignore
-    msalInstance.eventHandler.emitEvent(EventType.HANDLE_REDIRECT_START, InteractionType.Redirect);
+    msalInstance.controller.getEventHandler().emitEvent(EventType.HANDLE_REDIRECT_START, InteractionType.Redirect);
     // @ts-ignore
```

Comment:
remind me what the ts-ignore was here for?

Reply:
Got it thanks! It works because protected only exists in TypeScript-world, which you're ignoring